app/models/product.rb

```diff
@@ -0,0 +1,2 @@
+class Product < ActiveRecord::Base
```

Comment:
right!! . product should have reference of m_book. but I'll implement request model so ref of user should be written in request!!